src/componentes/grid-items.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import Brown_coffee_card from './coffee-card';
+import Blue_coffee_card from './coffee-card';
+
+class Grid_items extends Component {
+    state = {  }
+    render() { 
+        return ( 
+            <div className="grid-items">
+                <Brown_coffee_card />
+                <Brown_coffee_card />
+                <Brown_coffee_card />
+                <Brown_coffee_card />
+                <Blue_coffee_card />
+                <Blue_coffee_card />
+                <Blue_coffee_card />
+                <Blue_coffee_card />
```

Comment:
Use map and send items as a props 

```suggestion
             this.items.map ....   <Brown_coffee_card />
```